src/amber/dsl/router.cr

```diff
@@ -15,12 +20,24 @@ module Amber::DSL
       router.add(%route)
     end
 
-    {% for verb in {:get, :post, :put, :delete, :options, :head, :trace, :connect} %}
+    {% for verb in RESOURCES %}
 
       macro {{verb.id}}(*args)
         route {{verb}}, \{{*args}}
       end
 
     {% end %}
+
+    # TODO Clean this up
```

Comment:
Would like to know if there is a better implementation for this. My macro skills are limited. I tried approaching it with a nice hash and do the lookup base on the items in the actions variable.